Gems/ROS2/Code/Source/Georeference/GNSSFormatConversions.h

```diff
@@ -13,33 +13,36 @@
 namespace ROS2::GNSS
 {
 
+    AZStd::array<double,3> FromAzVector3(const AZ::Vector3& vector3);
```

Comment:
Please include type alias for WSG84 coordinates vector in a separate header so that it is universally used within the codebase - this is prone to issues

Reply:
Switched to custom struct here.